docs/source/how_to_guides/adding_a_new_model.rst

```diff
@@ -0,0 +1,198 @@
+Adding a New Model to Anomalib
+==============================
+New models can be added to anomalib and are always welcome. To add new models, the following steps should be followed:
```

Comment:
> New models can be added to anomalib and are always welcome

Maybe this could be rephrased. To prevent cluttering the library, I think we would only be interested in adding a model to the main branch if it's accompanied by a published paper. Or if we've tested it and decide it solves a specific problem or fills a gap in the library.